hack/alicloud/required-images.txt

```diff
@@ -0,0 +1,31 @@
+# Kops
+kope/dns-controller:1.15.0-beta.1
+kope/kops-controller:1.15.0-alpha.1
+
+# Kubernetes 1.16.4
+gcr.io/google_containers/kube-apiserver:v1.16.4
```

Comment:
I'm wondering how we'll keep these tags up to date. Perhaps `mirror.sh` could take a k8s version parameter? It could be used with these gcr.io/google_containers but the remaining images might be more tricky.

Perhaps a notice that says this list isn't necessarily up to date and the user is responsible for determining all image tags they need